### PR TITLE
added messageBridge feature

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208754077928486/f

## Description

- [x] added `messageBridge` feature
- [x] 🚧 ~added a macos example, that should be removed before we take away the 'DO NOT MERGE' label~ (removed)

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#2465** <a href="https://app.graphite.dev/github/pr/duckduckgo/privacy-configuration/2465?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: schema/shared.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface DomainLookup {
+  domain: string
```
Comment (jonathanKingston) [inline schema/shared.ts:2]:
```suggestion
  domain: string | string[]
```

Comment (shakyShane) [reply schema/shared.ts:2]: Does patchSettings support arrays then?

File: overrides/macos-override.json [hunk 2]
```diff
@@ -135,6 +135,24 @@
...
+                            {
+                                "op": "replace",
+                                "path": "/aiChat",
+                                "value": "enabled"
+                            }
+                        ]
+                    }
+                ]
+            }
+        },
```
Comment (shakyShane) [inline overrides/macos-override.json:155]: An example of enabling the `aiChat` feature

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**